tests/shared/drive-constraints.spec.js

```diff
@@ -556,4 +556,229 @@ describe('Shared: DriveConstraints', function() {
 
   });
 
+  describe('.getDriveImageCompatibilityStatus', function() {
+
+    beforeEach(function() {
+      if (process.platform === 'win32') {
+        this.mountpoint = 'E:\\foo';
+        this.separator = '\\';
+      } else {
+        this.mountpoint = '/mnt/foo';
+        this.separator = '/';
+      }
+
+      this.drive = {
+        device: '/dev/disk2',
+        name: 'My Drive',
+        protected: false,
+        system: false,
+        mountpoints: [
+          {
+            path: this.mountpoint
+          }
+        ],
+        size: 4000000000
+      };
+    });
+
+    describe('given there are no errors or warnings', () => {
+
+      it('should return OK', function() {
+        const result = constraints.getDriveImageCompatibilityStatus(this.drive, {
+          path: '/mnt/disk2/rpi.img',
+          size: 1000000000
+        });
+
+        m.chai.expect(result).to.deep.equal({
+          type: constraints.COMPATIBILITY_STATUS_TYPES.OK,
+          message: constraints.COMPATIBILITY_STATUS_MESSAGES.OK
+        });
+      });
```

Comment:
In all these test cases there is a blank between `describe` and `it`. Can we add a blank line here as well for symmetric purposes?